### ISSUE TITLE
Fatal error - require_once 'fontlib/src/FontLib/Table/Type/fftm.php'

### ISSUE THREAD

**Issue by consolibyte** [dompdf/php-font-lib#48]
I am getting this error when trying to render a document using DomPDF. 

require_once(/usr/home/app/www/html/signup/../library/fontlib/src/FontLib/Table/Type/fftm.php): failed to open stream: No such file or directory

Is this font type not supported, or?
##

**Comment by bsweeney** [comment]
What font are you using? The only thing I could find related to FFTM is a custom FontForge table. Since this is a custom table it's probably not supported by php-font-lib. @Phenx could better answer but I guess some exception handling around table classes might be in order to not fail on an unsupported class.

If this is indeed an issue related to a font created with FontForge I'd look to see if you can exclude that table. And if you can't do that maybe just craft a class to handle this table. If you look at the [head table class](https://github.com/PhenX/php-font-lib/blob/master/src/FontLib/Table/Type/head.php) you can see that not much is required. I haven't tested, but maybe even something as simple as the following would work:

```
<?php
namespace FontLib\Table\Type;
use FontLib\Table\Table;

class fftm extends Table {
}
```

**Comment by bsweeney** [comment]
I believe this was addressed by 04b2cf6d

**Comment by acobarras** [comment]
tengo el mismo error al implementar un checkbox o un radio en el pdf de DOMPDF no se que pueda ser

<b>Fatal error</b>:  Uncaught Error: Failed opening required 'C:\xampp\htdocs\sidpa_2\app/FontLib/Table/Type/fftm.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\sidpa_2\app\autoload.php:7
Stack trace:
